src/lib/dates.ts

```diff
@@ -129,28 +128,22 @@ export const dateRangeToString = (dateRange: DateRange, locale?: LocaleIdentifie
   }
 };
 
-/* istanbul ignore next */
-export const relativeDateToString = (
-  relativeDate: RelativeDate,
-  suffixes = { before: 'ago', after: '' },
-): string => {
-  const duration: string | undefined = DEFAULT_DURATIONS.find(
-    d => d.value === relativeDate.duration,
-  )?.label;
-  const suffix = ' ' + (relativeDate.quantity < 0 ? suffixes.before : suffixes.after);
-  return `${Math.abs(relativeDate.quantity)} ${duration?.toLowerCase()}${suffix.trimEnd()}`;
-};
-
 /* istanbul ignore next */
 export const relativeDateRangeToString = (
   relativeDateRange: RelativeDateRange,
   availableVariables: VariablesBucket = [],
   variableDelimiters: VariableDelimiters = { start: '', end: '' },
 ): string => {
   const identifier = extractVariableIdentifier(relativeDateRange.date, variableDelimiters);
-  const baseDateLabel = availableVariables.find(v => v.identifier === identifier)?.label;
-  const relativeDate = _pick(relativeDateRange, ['quantity', 'duration']);
-  const relativeDateLabel = relativeDateToString(relativeDate, { before: '', after: '' });
+  const baseDateLabel =
+    availableVariables.find(v => v.identifier === identifier)?.label ?? identifier;
+  const duration: string | undefined = DEFAULT_DURATIONS.find(
+    d => d.value === relativeDateRange.duration,
+  )?.label;
+  const suffix = ' ' + (relativeDateRange.quantity < 0 ? 'ago' : '');
```

Comment:
I think the "ago" suffix would only apply when talking from a base date of "now". So it's best to remove this logic IMHO.
("3 days ago before today" isn't correct I think, but "3 days before today" is)

Reply:
Yes indeed, we should do it in another PR with the removal of CUSTOM_DATE_RANGE_LABEL_SEPARATOR